src/koans/26_transducers.clj

```diff
@@ -0,0 +1,23 @@
+(ns koans.26-transducers
+  (:require [koan-engine.core :refer :all]))
+
+(def xfms
+  (comp (map inc)
+  (filter even?)))
```

Comment:
Can we indent this line or move it up to the previous line?

Reply:
aha yes an emacs fail. Will fix.